config/gocdb_schema.xml

```diff
@@ -493,7 +493,7 @@
 	    <!-- negation regex - allow all chars except negated -->
         <!--Allows most characters, including whitespace. However, no trailing or leading whitespace permitted -->
         <!-- <regex>/^[^`'\"&lt;&gt;\s]+(\s*[^`'\"&lt;&gt;\s]+)*$/</regex> -->
-        <regex>/^[^`'\"&lt;&gt;\s]+(\s*[^`'\"&lt;&gt;\s]+)*$/</regex>
+        <regex>/^[^`\"&lt;&gt;\s]+(\s*[^`\"&lt;&gt;\s]+)*$/</regex>
```

Comment:
This now seems to be the same expression as is commented out on the line before. Do we know why the single quote was, presumably, deliberately removed?

Reply:
> This now seems to be the same expression as is commented out on the line before

Good spot!

> Do we know why the single quote was, presumably, deliberately removed?

Alas no, this was the commit that introduced it - https://github.com/GOCDB/gocdb/commit/037e16cad456d2df2e334e55b576c6137665e52d

Reply:
See https://github.com/GOCDB/gocdb/commit/83f38c4177c054ca30e85e308c4bb7ef4240bb39